hello-world/enclave/src/main/java/com/r3/conclave/sample/enclave/ReverseEnclave.java

```diff
@@ -0,0 +1,96 @@
+package com.r3.conclave.sample.enclave;
+
+import com.r3.conclave.enclave.Enclave;
+import com.r3.conclave.mail.EnclaveMail;
+import java.io.FileOutputStream;
+import java.io.FileInputStream;
+import java.io.File;
+import java.nio.charset.StandardCharsets;
+import java.util.Map;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+/**
+ * Simply reverses the bytes that are passed in.
+ */
+public class ReverseEnclave extends Enclave {
+    // We store the previous result to showcase that the enclave internals can be examined in a mock test.
```

Comment:
We have `ReverseEnclave` used in the Hello World example I think. Would it be nicer to call this class `ReversePersistentEnclave` or something similar, to give the idea that this is a different sample and to express the fact that we are not just reversing a string but actually persisting to files?
More directly, wouldn't be nicer to actually remove the concept of reversing a string, which is not related to persistent?
For example, using the string to actually use it to write and read into files/persistent map, for example?